launchscripts/launch.bat

```diff
@@ -14,6 +17,13 @@ SET MAX_RAM={{maxRAM}}
 
 :: DO NOT EDIT ANYTHING PAST THIS LINE
 SET LAUNCHPARAMS=-server -Xms%MIN_RAM% -Xmx%MAX_RAM% %JAVA_PARAMETERS% -jar %FORGEJAR% nogui
+echo Checking java version...
+echo.
+java -version
+echo.
+echo The expected java version is 1.8. Not higher, not lower.
+echo.
+
 echo Launching the server...
 echo.
 echo ^> java %LAUNCHPARAMS%
```

Comment:
Please also add the message from `launch.sh:38` before the `pause` command here for parity between scripts.

Reply:
Unnecessary, pause adds its own localized text.

Reply:
Ah, right. Forgot it does that. Nevermind then.